US-CBP/cbp-theme
# add component for code snippets

#### What's this PR do?
-Example implementation of the code markup component using prism.js for review by the ui dev & creative teams.

#### Where should the reviewer start?
-CodeSnippet component under `ds-components` folder
-`.babelrc.json` (config for prism.js)


#### How should this be manually tested?
- npm install dependencies in `ds-ux-guidelines`
- npm run css:build website:build
- navigate to the layout page under "Foundations" in the style guide
- hover over code snippet to see the copy button on the top right corner. 

#### Any background context you want to provide?
- changes in `typography.js` were mostly for testing purposes. After approval of the team I will be implementing the code snippets in there.
- The prop `language` used for CodeSnippet needs to be exact. List of languages + class name format found here: https://prismjs.com/#supported-languages


#### What are the relevant tickets?
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/13155212/78591934-36a8d480-7812-11ea-97d0-214ccc452f2e.png)


#### Questions:
- Is there a blog post?
- Does the knowledge base need an update?
